spec/book_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'rspec'
+require_relative '../Classes/book'
+
+describe Book do
+  before :each do
+    @book = Book.new('publisher', 'cover_state', '2020/2/3')
+  end
+
+  describe '#new' do
+    it 'should create a new book' do
+      expect(@book).to be_instance_of(Book)
+    end
+    it 'should throw an error without three arguments' do
+      expect { Book.new('Publisher', 'CoverState') }.to raise_error(ArgumentError)
+    end
+    it 'should have cover_state' do
+      expect(@book.cover_state).to eq('cover_state')
+    end
+    it 'should have publisher' do
+      expect(@book.publisher).to eq('publisher')
+    end
+    it 'should have cover_state settable' do
+      @book.cover_state = 'good'
+      expect(@book.cover_state).to eq('good')
+    end
+    it 'should not have publisher settable' do
+      expect { @book.publisher = 'new_guy' }.to raise_error NameError
+    end
```

Comment:
You can always have multiple `expect` for the same `it` which could save you a couple of lines of code.